docs/UserGuide.md

```diff
@@ -115,6 +115,10 @@ Other fields are **multi-valued fields**. Each contact has a **list** of differe
 | specialisations     | /spec   | Any non-empty value                                          |
 | tags                | /tag    | Any non-empty value                                          |
 
+Please take note:
```

Comment:
These additional notes seem slightly out of place. Graduation year and courses also have additional info and we should either put all in the table or put all below the table.

I feel it's better to put all in the table. It will look more organised to create another column called additional remark, so that the format column only contains the exact, succint command format, and any noteworthy remark can be put in the new column.

Reply:
Changed, please help me test if the new table renders properly with Jekyll.

Reply:
Tested and works properly.

Also formatted remarks for other fields in the 2 tables. Can help me check through and merge if there's no more change.